openrouteservice/src/main/java/org/heigit/ors/api/requests/routing/RouteRequest.java

```diff
@@ -178,8 +178,8 @@
     private boolean hasBearings = false;
 
     @ApiModelProperty(name = PARAM_CONTINUE_STRAIGHT,
-            value = "Forces the route to keep going straight at waypoints restricting uturns there even if it would be faster. This setting will work for all profiles except for `driving-*`. " +
-            "CUSTOM_KEYS:{'apiDefault':'true','validWhen':{'ref':'profile','valueNot':['driving-*']}}",
+            value = "Forces the route to keep going straight at waypoints restricting uturns there even if it would be faster." +
+            "CUSTOM_KEYS:{'apiDefault':'true'}",
```

Comment:
I know this is from before, but is this true? I thought that by default continue straight was disabled as it would force the use of slower algorithms.

Reply:
you're right, the actual default is false, corrected docs